libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackAdapter.cs

```diff
@@ -32,9 +32,9 @@ public class SlackAdapter : BotAdapter, IBotFrameworkHttpAdapter
         public SlackAdapter(ISlackAdapterOptions options)
             : base()
         {
-            this.options = options;
+            _options = options ?? throw new ArgumentNullException(nameof(options));
```

Comment:
can options be null?

Reply:
Yes, the options are sett by the user when creating the adapter.